c2corg_ui/templates/profile/data.html

```diff
@@ -14,46 +14,48 @@
 
 % else:
 
-  % if geometry:
-
-    % if profile.get('areas'):
-      <article class="location-static">
-        ${show_areas(profile)}
-      </article>
+  <div app-view-details>
+    % if geometry:
+
+      % if profile.get('areas'):
+        <article class="location-static">
+          ${show_areas(profile)}
+        </article>
+      % endif
+
+      <div ng-init='profileFeature = {
+          "type": "FeatureCollection",
+          "properties": {},
+          "features": [{
+            "type": "Feature",
+            "geometry": {"type": "Point", "coordinates": [${geometry.x}, ${geometry.y}]},
+            "properties": {
+              "title": ${dumps(profile["name"]) | n},
+              "lang": "${locale["lang"]}",
+              "documentId": ${profile['document_id']},
+              "module": "profiles",
+              "highlight": true
+            }
+          }]
+        };
+        detailsCtrl.documentService.setAssociations(${profile['associations']});'>
```

Comment:
This is the main change of this commit with the transfer of the ``app-view-details`` directive from the main template to this one (else the directive was initialized before the image associations were available. The rest of the change are simply reindents.